README.md

```diff
@@ -3,8 +3,10 @@
 ## Install
 
 ```console
+$ ./autogen.sh
+$ ./configure --prefix=/usr
 $ make
-$ mv slirp4netns ~/bin
+$ make install
```

Comment:
Maybe `--prefix=$HOME` should be default?

Reply:
s/default/recommended config/

Reply:
That will install the program under `$HOME/bin`.  Is that the preferred location?